styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
+  text-align: justify;
+  padding-top: 50px;
+  padding-bottom: 50px;
```

Comment:
Do you need both `padding-top` and `padding-bottom` here? `padding-top` should be sufficient to clear the navigation bar.